### PR TITLE
Greasemonkey script that adds a link to unread notifications in Phabricator

### PR DESCRIPTION
**Pull request by sebastian-berlin-wmse**
The link appears as a bullhorn next to the bell icon, if there are any
unread notifications.

### CODE REVIEW

File: phabricator.user.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// ==UserScript==
+// @name     Phabricator
+// @version  1.0
+// @grant    none
+// @require  https://code.jquery.com/jquery-3.4.1.slim.min.js
+// @include  https://phabricator.wikimedia.org/*
```
Comment (lokal-profil) [inline phabricator.user.js]: Add 
```suggestion
// @include  https://phabricator.wikimedia.org/*
// @downloadURL https://github.com/Wikimedia-Sverige/scripts/raw/master/phabricator.user.js
```

File: phabricator.user.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+$(function() {
+    $("<a></a>")
+        .attr("href", "/notification/query/unread/")
+        .addClass("alert-notifications")
+        .append(
+            $("<span></span>")
+                .addClass("phabricator-main-menu-message-icon phui-icon-view phui-font-fa fa-bullhorn")
+        )
+        .addClass("alert-notifications")
```
Comment (lokal-profil) [inline phabricator.user.js]: This is a duplicate line which can be removed
```suggestion
```

File: phabricator.user.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+// @include  https://phabricator.wikimedia.org/*
+// ==/UserScript==
+
+$(function() {
+    $("<a></a>")
+        .attr("href", "/notification/query/unread/")
+        .addClass("alert-notifications")
+        .append(
+            $("<span></span>")
+                .addClass("phabricator-main-menu-message-icon phui-icon-view phui-font-fa fa-bullhorn")
```
Comment (lokal-profil) [inline phabricator.user.js]: The following changes the icon to have the same white colour as the bell.
```suggestion
        .addClass("alert-notifications alert-unread")
        .append(
            $("<span></span>")
                .addClass("phabricator-main-menu-alert-icon phui-icon-view phui-font-fa fa-bullhorn")
```

**Review by lokal-profil — CHANGES_REQUESTED** [review]
Trying the fancy new suggestion feature which should make it easier to add the changes. Sadly doesn't work for the multiline one.

File: phabricator.user.js (hunk 3 above)
Comment (lokal-profil) [reply phabricator.user.js]: This cannot be automatically applied (unlike the above ones) :(

File: phabricator.user.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+// @version  1.0
+// @grant    none
+// @require  https://code.jquery.com/jquery-3.4.1.slim.min.js
+// @include  https://phabricator.wikimedia.org/*
+// ==/UserScript==
+
+$(function() {
+    $("<a></a>")
+        .attr("href", "/notification/query/unread/")
+        .addClass("alert-notifications")
```
Comment (lokal-profil) [inline phabricator.user.js]:
```suggestion
        .addClass("alert-notifications alert-unread")
```

File: phabricator.user.js (hunk 3 above)
Comment (lokal-profil) [inline phabricator.user.js]:
```suggestion
                .addClass("phabricator-main-menu-alert-icon phui-icon-view phui-font-fa fa-bullhorn")
```

File: phabricator.user.js (hunk 1 above)
Comment (sebastian-berlin-wmse) [reply phabricator.user.js]: Added.

File: phabricator.user.js (hunk 3 above)
Comment (sebastian-berlin-wmse) [reply phabricator.user.js]: Fixed.

**Review by lokal-profil — APPROVED** [review]

**Pull request merged**